packages/components/app/styles/components/app-footer.scss

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// app-footer
+//
+
+// WIP: names, etc. may change
+// :root {
+//   --token-app-footer-text-color: var(--token-color-foreground-faint);
+//   --token-app-footer-border-color: var(--token-color-border-primary);
+// }
+
+$app-footer-gap: 24px;
+
+.hds-app-footer {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $app-footer-gap;
+  justify-content: flex-end;
+  padding: 24px;
+  border-top: 1px solid;
+}
+
+.hds-app-footer__list,
```

Comment:
To discuss: you can have only `hds-app-footer__list` here (and maybe add `hds-app-footer__list--legal-links` but only for the highlighting?)

Reply:
discussed, OK to leave it as is